<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos de la vivienda sin garaje con un tipo de cubierta más atrasada a su construcción</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="boton_mostrar">
        <property name="text">
         <string>Mostrar Datos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boton_cerrar">
        <property name="text">
         <string>Cerrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Escoja el Tipo de techo</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="tipo_de_techo">
        <item>
         <property name="text">
          <string>placa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>teja de zinc</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>teja de asbesto cemento</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>teja de arcilla</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTableWidget" name="tabla_de_datos"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
